packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -122,55 +122,6 @@ $ms-Persona-presenceBorder: 2px;
     // Using important to override all of the more specific initialsIsColor rules
     background-color: Window !important;
   }
-
-  &.initialsIsLightBlue {
```

Comment:
It looks like the variables from line 12-27 will be obsolete now, I can't find references to them anywhere else. I think it's safe to remove them.

Reply:
Nice catch! Removing.

Reply:
I removed the unused sass. Anymore feedback @lynamemi?